BinaryTreeLevelOrderTraversal/Solution.java

```diff
@@ -0,0 +1,93 @@
+/**
+ * Definition for a binary tree node.
+ * public class TreeNode {
+ *     int val;
+ *     TreeNode left;
+ *     TreeNode right;
+ *     TreeNode() {}
+ *     TreeNode(int val) { this.val = val; }
+ *     TreeNode(int val, TreeNode left, TreeNode right) {
+ *         this.val = val;
+ *         this.left = left;
+ *         this.right = right;
+ *     }
+ * }
+ */
+
+// Time Complexity : O(n)
+// Space Complexity : O(log n)
```

Comment:
Check space complexity. Height of a complete binary is log n and no where in question its mentioned that it is a complete binary tree